composer.json

```diff
@@ -18,13 +18,14 @@
     "require": {
         "php": ">=5.3.2",
         "swiftmailer/swiftmailer": ">=4.2.0,~5.0",
-        "symfony/swiftmailer-bridge": "~2.1"
+        "symfony/swiftmailer-bridge": "~2.3"
     },
     "require-dev": {
-        "symfony/dependency-injection": "~2.1",
-        "symfony/http-kernel": "~2.1",
-        "symfony/config": "~2.1",
-        "symfony/yaml": "~2.1"
+        "symfony/dependency-injection": "~2.3|~3.0",
+        "symfony/http-kernel": "~2.3|~3.0",
+        "symfony/config": "~2.3|~3.0",
```

Comment:
some of these should actually become requirements IMO: this bundle is only about configuring Swiftmailer in the framework, so it cannot be used without them